tests/unit/handlers/test_web.py

```diff
@@ -0,0 +1,130 @@
+import unittest
```

Comment:
Should we also add some tests for `CrawlerTable`? For it's `select()` operation to be precise? Such as for correctly raising exceptions when a OR operator is included or when an invalid URL is passed?

Reply:
Yes, I will add them